src/algorithm/storage/storage.rs

```diff
@@ -0,0 +1,73 @@
+use serde_json::{Value, Map};
+use std::fs;
+use std::path::Path;
+use std::error::Error;
+///
+/// Класс, для хранения данных из json файла
+/// - 'file_path' - путь к файлу json
+/// - 'data' - данные json файла
+pub struct Storage {
+    file_path: String,
+    data: Value,
+}
+//
+//
+impl Storage {
+    ///
+    /// Конструктор класса
+    /// - 'file_path' - путь к файлу json
+    pub fn new(file_path: &str) -> Result<Self, Box<dyn Error>> {
+        let data = if Path::new(file_path).exists() {
+            let content = fs::read_to_string(file_path)?;
+            serde_json::from_str(&content)?
+        } else {
+            Value::Object(Map::new())
+        };
+        Ok(Storage {
+            file_path: file_path.to_string(),
+            data,
+        })
+    }
+    ///
+    /// Метод сохранения изменений в файл json
+    pub fn save(&self) -> Result<(), Box<dyn Error>> {
+        let content = serde_json::to_string_pretty(&self.data)?;
+        fs::write(&self.file_path, content)?;
+        Ok(())
+    }
+    ///
+    /// Метод получения данных
+    /// - 'path' - путь к данным
+    pub fn get(&self, path: &str) -> Option<&Value> {
```

Comment:
rename into `load(key: &str)` as specified in the issue
if `store` returns `Result`, use result in this case also